src/clients/googleClassroom.js

```diff
@@ -0,0 +1,9 @@
+const BASE_URL = 'https://classroom.google.com/u/0/share?url=';
+
+export function createShareToClassroomUrl(snapshotKey) {
+  const uri = document.createElement('a');
+  uri.setAttribute('href', '/');
+  uri.search = `snapshot=${snapshotKey}`;
+  const classroomShareUrl = BASE_URL + uri.href;
```

Comment:
This won’t properly escape the URL being used in the query parameter—better to remove the `uri=` from the `BASE_URL`, and then do `classroomShareUrl = BASE_URL + qs.stringify({url: uri.href})` ([`qs` reference here](https://www.npmjs.com/package/qs); it’s already a dependency)